host-configs/arcticus-apmidg-oneapi.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019-2022 Lawrence Livermore National Security, LLC and other
+# Variorum Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+# Only tested on JLSE login nodes:
+#
+# oneapi 2022.06.30.002
+# apmidg 0.2.0
+#
+# First, login in to a JLSE login node
+# $ module load oneapi apmidg jansson hwloc libiconv cmake
+# $ git clone https://github.com/anlsys/variorum.git && cd variorum
+# $ mkdir build && cd build
+# $ cmake -C ../host-configs/arcticus-apmidg-oneapi.cmake -DCMAKE_INSTALL_PREFIX=__INST_DIR__ ../src
+#
+# Please contact Kazutomo Yoshii <kazutomo@mcs.anl.gov>
+# if you have a question on Intel DGPU support.
+
+set(ENABLE_FORTRAN OFF CACHE PATH "")
+
+# build for specific architecture
+set(VARIORUM_WITH_AMD OFF CACHE BOOL "")
+set(VARIORUM_WITH_ARM OFF CACHE BOOL "")
+set(VARIORUM_WITH_IBM OFF CACHE BOOL "")
+set(VARIORUM_WITH_INTEL OFF CACHE BOOL "")
+set(VARIORUM_WITH_INTEL_DGPU ON CACHE BOOL "")
+set(VARIORUM_WITH_NVIDIA OFF CACHE BOOL "")
```

Comment:
@slabasan @kazutomo 

Should be VARIORUM_WITH_NVIDIA_GPU here.

VARIORUM_WITH AMD_GPU is also missing.

Both those flags are off and don't matter in the build, but important for consistency across Variorum